11-moving-to-your-own-machine.Rmd

```diff
@@ -11,10 +11,31 @@ you don't have a preconfigured JupyterHub available to you.
 
 By the end of the chapter, students will be able to:
 
+- use a Jupyter notebook to execute provided R code
```

Comment:
We can actually remove all of this Jupyter stuff from this chapter, as I made a new chapter that includes all of this in `getting-started-with-jupyter`. I think if we remove this, merging will be easier.